partiql-planner/src/testFixtures/kotlin/org/partiql/planner/test/Parsing.kt

```diff
@@ -127,7 +128,7 @@ public fun StructElement.toStructType(): StaticType {
         val type = it.getAngry<IonElement>("type").toStaticType()
         StructType.Field(name, type)
     }
-    return StructType(fields)
+    return StructType(fields, contentClosed = true, constraints = setOf(TupleConstraint.Open(false)))
```

Comment:
Latest in `main` has handling of constraints.

https://github.com/partiql/partiql-lang-kotlin/blob/main/plugins/partiql-local/src/main/kotlin/org/partiql/plugins/local/LocalSchema.kt#L125-L139